### PR TITLE
Basic Views are set up

### PR DESCRIPTION
**Pull request by MartyTheeMartian**
* Fixed Issues from previous PR
* Used Pursuance Image for Home Page
* Referenced local files instead of cdn

### CODE REVIEW

**Comment by MartyTheeMartian** [comment]
@elimisteve New & Improved PR

**Comment by elimisteve** [comment]
@MartyTheeMartian Great! Looking...

File: public/index.html [hunk 1]
```diff
@@ -1,40 +1,22 @@
...
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>React App</title>
+    <link rel="stylesheet" href="%PUBLIC_URL%/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="font-awesome.min.css/font-awesome.min.css"/>
```
Comment (elimisteve) [inline public/index.html]: @MartyTheeMartian I assume this `href` should use `%PUBLIC_URL%` as well?

**Review by elimisteve — COMMENTED** [review]
@MartyTheeMartian Do public/css/{font-awesome,font-awesome}.min.css need to be in the repo?

Also, I now see that what has always been called the `static/` directory is what React now calls `public/`.  Want to move things from `static/` to `public/`, or should I?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Please fork and clone down this repository to your local machine.
+
+If you don't have React.js globally installed, run this command in your command line:
+
+```
+npm install -g create-react-app
+```
+
+Next, change directory into the `pursuance` folder.
+Run this this command to install node modules:
```
Comment (elimisteve) [inline README.md]: @MartyTheeMartian `this this`

File: public/index.html (hunk 1 above)
Comment (MartyTheeMartian) [reply public/index.html]: ah yes

**Comment by MartyTheeMartian** [comment]
@elimisteve only way I could reference a css or js file in the HTML Doc was in the public folder, webpack bundles everything else

**Comment by MartyTheeMartian** [comment]
@elimisteve This PR is a fail ... new one coming in

**Pull request closed without merging**